.github/workflows/stale.yml

```diff
@@ -14,12 +14,12 @@ jobs:
         with:
           stale-issue-message: 'This issue was marked stale due to lack of activity and will be closed in 7 days. Commenting will instruct the bot to automatically remove the label. This bot runs once per day.'
           close-issue-message: 'Closed as inactive. Feel free to reopen if this issue is still a concern.'
-          stale-pr-message: 'This PR was marked stale due to lack of activity and will be closed in 7 days. Commenting or Pushing will instruct the bot to automatically remove the label. This bot runs once per day.'
+          stale-pr-message: 'This PR was marked stale due to lack of activity and will be closed in 7 days. Commenting or pushing will instruct the bot to automatically remove the label. This bot runs once per day.'
           close-pr-message: 'Closed as inactive. Feel free to reopen if this PR is still being worked on.'
           operations-per-run: 400
           days-before-pr-stale: 7
           days-before-issue-stale: 1200
           days-before-pr-close: 7
           days-before-issue-close: 7
           exempt-all-issue-milestones: true
-          exempt-issue-labels: need-runtime-change,await-spec,help wanted,needs-triage
```

Comment:
@TimothyMothra @cijothomas @rajkumar-rangaraj @vishweshbankwar 

I spent some time today triaging issues. Decided to make this list only `needs-triage` because we already exclude anything assigned to a milestone. It felt like the best thing to do was use "Future" milestone as a backlog for anything accepted after triage.

This is the triage process I sort of landed on doing the exercise...

* Triager should review the feature request/bug/question.
* Add missing labels.
* Discuss as needed.
* Make a decision...
  * Close the issue if question is answered, issue is duplicate, issue is not valid, or whatever.
  * If accepting the issue/feature request:
     * Add to "Future" milestone anything which is accepted but not committed. This could be because it needs a spec change or runtime change or it is just something we may or may not do in the future. Also things "help wanted" or "good first issue" go here.
     * Add to a specific release milestone anything which is accepted & committed to be done.

Thoughts?

Reply:
Regarding the question, I think we should remove the questions option from issues. We should direct users to discussions for that.

Reply:
If the goal is to maintain a large backlog, I don't have a problem with this.

But if the goal is to trim the fat, I would caution though that "Future" is kind of a generic catch all. I felt the "need-runtime-change", "await-spec", and "help-wanted" were fairly specific. There's a risk that if "Future" is used too frequently it becomes a bucket where issues go and never expire.

But as this is all a new processes, lets give it a try and we can continue to evolve as needed. :)

Reply:
@vishweshbankwar 

> Regarding the question, I think we should remove the questions option from issues. We should direct users to discussions for that.

We do kind of do that already: https://github.com/open-telemetry/opentelemetry-dotnet/blob/0e011e8c3c2e668c87581386e30624885536f32b/.github/ISSUE_TEMPLATE/question.yml?plain=1#L9-L10

Still seems like we get a lot of question issues though 🤣 

There seems to be a GitHub feature to automatically convert things to discussions based on labels: https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#converting-issues-based-on-labels

I thought about clicking it but confusing messaging around how it works: https://github.com/orgs/community/discussions/52785

I decided to leave it alone for now but it is something we should definitely look at!

@TimothyMothra 

> But as this is all a new processes, lets give it a try and we can continue to evolve as needed. :)

Agree. I doubt this is perfect but we can continue to refine it!

Some other thoughts...

> But if the goal is to trim the fat, I would caution though that "Future" is kind of a generic catch all.

I thought about creating other milestones. Like "NotPlanned" or "Blocked" but decided to start small 🤷

> I felt the "need-runtime-change", "await-spec", and "help-wanted" were fairly specific. 

What I found trying to triage things is there are lot of states. Some issues are specifically left out there to gather feedback. Sometimes we do announcements for known issues or bugs. What I found was the list of excluded labels kept growing on me which I thought was silly. I also didn't want to create a bunch of new labels. Just feeling it out milestones seemed to work better.

> There's a risk that if "Future" is used too frequently it becomes a bucket where issues go and never expire.

For sure! Check out runtime: https://github.com/dotnet/runtime/milestone/1. 7,089 issues in Future at the moment 🤣

Reply:
> 7,089 issues in Future at the moment

😭